packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -682,18 +684,15 @@ describe("BlobManager", () => {
 			await runtime.attach();
 			await runtime.connect();
 			const ac = new AbortController();
-			let handleP;
+			const blob = IsoBuffer.from("blob", "utf8");
+			const handleP = runtime.createBlob(blob, ac.signal);
+			ac.abort("abort test");
+			assert.strictEqual(runtime.unprocessedBlobs.size, 1);
 			try {
-				const blob = IsoBuffer.from("blob", "utf8");
-				handleP = runtime.createBlob(blob, ac.signal);
-				ac.abort("abort test");
-				assert.strictEqual(runtime.unprocessedBlobs.size, 1);
 				await runtime.processBlobs(false);
```

Comment:
this is part of the test harness, it should not fail, or if it is expected to fail the failures are not relevant to system under test. the only errors that matter are errros from our public apis, to handleP in this case.

Reply:
this comment applies generally to this PR, there are a number of places where the validation is no longer validating the product code, its just test code validating itself.

Reply:
The relevant parts in this test are the validations after the catch I believe, ensuring the handle rejected and having a clean summary which I do in other tests as well.  processBlobs is expected to fail, since it's provoking a rejection at the storage level. Which validations I'm misising here or how can I deal with the expected failures better?

Reply:
i don't think process blobs should fail as a blob failing to upload is still a successful processing

Reply:
This comment left me thinking about what will happen when upload fails without aborting. We are currently throwing an error so I made some changes and add a test as well. Probably these changes will be removed once my other PR gets merged but just for consistency reasons and later test validation I'll add them. 

Once we use runWithRetry we will need to make a distiction between retriable and not retriable errors.